src/components/HomeCard/index.js

```diff
@@ -0,0 +1,259 @@
+import React, { useState } from "react";
+import Slider from 'react-input-slider';
+import style from "./styles";
+
+
+const HomeCard = ({ className }) => {
+    const [valor, setState] = useState({ x: 0 });
```

Comment:
"x" is generally not a good variable/attribute name. Try using names that enable developers to understand the purpose of said variable just by reading the code.